app/services/by_team_project_fetcher_service.rb

```diff
@@ -0,0 +1,35 @@
+class ByTeamProjectFetcherService
+  def initialize(user)
+    @team = user.team
+  end
+
+  def in_progress
+    return [] if @team.nil?
+
+    if @team.eql?("regional_casework_services")
+      Project.assigned_to_regional_caseworker_team.in_progress.includes(:assigned_to)
```

Comment:
We are going to need to sort all these by `conversion_date` which means they all need to be `Conversion::Project`.